Add fortitude instructions (#607)
MetOffice/simulation-systems: source/FurtherDetails/fortitude_linter.rst

auto-fix options, and a summary of the repositories with errors at the bottom of the report.

When you have resolved the lint issue, see the instructions below on how to
re-test quickly.

N.B. Some errors can be :ref:`fixed automatically <auto-fix>` by fortitude.
This is rule dependent and Fortitude will let you know if the specific failure has this
option under each error listed.

Testing with Test Suite:
------------------------

Run these commands from the inside the top directory of the codebase:

- You can re-run fortitude_linter individually:

.. code-block:: shell

    cylc vip -z group=fortitude_linter -n suite_name ./rose-stem

- Or with scripts group:

.. code-block:: shell

    cylc vip -z group=scripts -n suite_name ./rose-stem

Testing manually from the terminal:
----------------------------------------

The following commands run fortitude on specific files and specify the rule or
configuration file.
Fortitude can be run without specifying these things but it may pick up existing
errors in the codebase that are unrelated to your changes.

The rule code associated with the error is given in the job.err report.

> If the repository with the error uses the universal configuration

Reviewer: ```suggestion
For the LFRic repositories, if the application with the error uses the universal configuration
```

(*lfric_apps/rose-stem/app/check_fortitude_linter/file/fortitude.toml*)
for its fortitude tests, the job.out specifies that in that repo’s output
section.
Otherwise, it will use its own configuration file, located in its top
directory e.g. *lfric_apps/lfricinputs/fortitude.toml*.

Run the following commands from the inside the top directory of the codebase:

> **First, load the lfric environment:**

Reviewer: ```suggestion
**First, load an environment with fortitude available**
```

.. code-block:: shell

>     ml use ~lfricadmin/lmod

Reviewer: ```suggestion
    # eg. For Met Office VDI's
    ml use ~lfricadmin/lmod
```
